scalastyle-config.xml

```diff
@@ -9,8 +9,19 @@
     <check level="warning" class="org.scalastyle.file.HeaderMatchesChecker" enabled="true">
         <parameters>
             <parameter name="header"><![CDATA[/*
- * Copyright 2023 HM Revenue & Customs
+ * Copyright 2024 HM Revenue & Customs
```

Comment:
Shouldn't we be using short format for perf repo? just comparing to some of our others

Reply:
Repo is still public so leaving til change